src/main/java/net/dv8tion/jda/core/handle/MessageUpdateHandler.java

```diff
@@ -52,17 +52,17 @@ protected Long handleInternally(JSONObject content)
                 switch (type)
                 {
                     case DEFAULT:
-                        return handleDefaultMessage(content);
+                        return handleMessage(content);
                     default:
                         WebSocketClient.LOG.debug("JDA received a message of unknown type. Type: {} JSON: {}", type, content);
                         return null;
                 }
             }
             else
             {
-                //TODO: handle partial message update info. Example:
-                //From webhook/rich-embed.
-                //{"author":{"bot":true,"id":"233501884294365184","avatar":"27ae7496b3b30cddab2feaaed06d862a","username":"GitHub","discriminator":"0000"},"id":"234838126969880596","embeds":[{"color":15109472,"author":{"icon_url":"https://avatars.githubusercontent.com/u/2415829?v=3","name":"abalabahaha","proxy_icon_url":"https://images-ext-2.discordapp.net/eyJ1cmwiOiJodHRwczovL2F2YXRhcnMuZ2l0aHVidXNlcmNvbnRlbnQuY29tL3UvMjQxNTgyOT92PTMifQ.mMTBuOMUKYowcUU1H8Gzc7g4fFQ","url":"https://github.com/abalabahaha"},"description":"It was removed from the unofficial docs and other places because devs didn't want automated registration.","type":"rich","title":"[hammerandchisel/discord-api-docs] New comment on issue #148: Register API call","url":"https://github.com/hammerandchisel/discord-api-docs/issues/148#issuecomment-252524279"}],"channel_id":"168311874624946176"}
+//                WebSocketClient.LOG.warn("Received MESSAGE_UPDATE with no specified \"type\" field. JSON: " + content);
```

Comment:
Same as above.

Reply:
This is commented out as it isn't going to be logged.

Reply:
:man_facepalming: